src/lib/analytics.js

```diff
@@ -138,25 +138,64 @@ store.subscribe(({isSignedIn}) => {
   ga('set', dimensions.SIGNED_IN, isSignedIn ? '1' : '0');
 });
 
-/**
- * Add a listener to detect back/forward cache restores and track them
- * as pageviews with the "bfcache" navigation type set (in case we need
- * to distinguish them from regular pageviews).
- * https://web.dev/bfcache/#how-bfcache-affects-analytics-and-performance-measurement
- */
-window.addEventListener(
-  'pageshow',
+function sendPageView() {
   /**
-   * @param {PageTransitionEvent} e
+   * Add a listener to detect back/forward cache restores and track them
+   * as pageviews with the "back-forward-cache" navigation type set (in
+   * case we need to distinguish them from regular pageviews).
+   * https://web.dev/bfcache/#how-bfcache-affects-analytics-and-performance-measurement
    */
-  (e) => {
-    if (e.persisted) {
-      ga('set', dimensions.NAVIGATION_TYPE, 'back-forward-cache');
-      ga('send', 'pageview');
-    }
-  },
-);
+  window.addEventListener(
+    'pageshow',
+    /**
+     * @param {PageTransitionEvent} e
+     */
+    (e) => {
+      if (e.persisted) {
+        ga('set', dimensions.NAVIGATION_TYPE, 'back-forward-cache');
+        ga('send', 'pageview');
+      }
+    },
+  );
+
+  // Handle prerenders that have already activated
+  if (
+    performance &&
+    performance.getEntriesByType &&
+    performance.getEntriesByType('navigation')[0].activationStart > 0
+  ) {
+    ga('set', dimensions.NAVIGATION_TYPE, 'prerender');
+    ga('send', 'pageview');
+    return;
+  }
+
+  // Handle prerenders that activate in the future
+  if (document.prerendering) {
+    document.addEventListener(
+      'prerenderingchange',
+      () => {
+        ga('set', dimensions.NAVIGATION_TYPE, 'prerender');
+        ga('send', 'pageview');
+      },
+      {once: true},
+    );
+    return;
+  }
+
+  // Handle normal page views
+  try {
```

Comment:
This try/catch was added in the template to handle cases where `performance.getEntriesByType()` may not be supported by the current browser. If you use the logic to determine navigation type I suggest above, then a try/catch is probably not necessary.